### ISSUE TITLE
Export command failed with "tar format cannot archive socket"

### ISSUE THREAD

**Issue by kraml** [DDoSolitary/LxRunOffline#75]
When I tried to export it failed with below error message:

```[ERROR] Error occurred while processing the archive: tar format cannot archive socket```

It is because I have some socket files create by gpg-agent under ~/.gnupg. Can this error be ignored and move on to finish the export process?

**Comment by Biswa96** [comment]
Is there any background process which is using that "file"?

**Comment by kraml** [comment]
No. I killed gpg-agent process and quite the whole WSL before exported the fs.

**Comment by DDoSolitary** [comment]
@kraml Sorry for the delay. Successfully reproduced, working on it

**Comment by kraml** [comment]
Absolutely no need for sorry, thank you for this great tool and fixing of the bug.

**Comment by kraml** [comment]
Yes I saw warning of ignoring unsupported files and the export process kept running until finished. Thanks!
